.github/CODEOWNERS

```diff
@@ -4,14 +4,14 @@
 # precedence. When someone opens a pull request that only
 # modifies JS files, only @js-owner and not the global
 # owner(s) will be requested for a review.
-
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence,
-# @forcedotcom/pdt will be requested for
+# @forcedotcom/ide-experience will be requested for
 # review when someone opens a pull request.
-*       @forcedotcom/pdt
+*       @forcedotcom/ide-experience
 
 # If the change occurs within the docs directory, add the doc maintainers.
+
 /docs/ @forcedotcom/doc-maintainers
 #ECCN:Open Source
-#GUSINFO: Platform Dev Tools Scrum Team, Platform Dev Tools
+#GUSINFO: Platform Dev Tools Scrum Team, IDE Experience Team
```

Comment:
Not sure what this does but figured I should update it since Platform Dev Tools doesn't exist any longer.  the scrum team name is still the same

Reply:
So in Gus we're labeled as "IDE Experience Team" now, but the technical label is still Platform Dev Tools Scrum Team?

Reply:
I went and looked at the git2gus page and it doesn't say anything about the code owners file.  So I'm no sure if this is a relic from the days before git2gus had it's own config file or what.  I think the intention is 
Platform Dev Tools Scrum Team -> person (this is the default user we have on WIs)
IDE Experience Team -> team